API_REFERENCE.md

```
@@ -0,0 +1,307 @@
+- `400 Bad Request`: 입력 데이터 오류
+- `401 Unauthorized`: 인증 실패
+
+### 게시글 목록 조회
+게시글 목록을 페이징하여 조회합니다.
+
+```http
+GET /api/posts
+```
+
+#### Query Parameters
+| Parameter | Type | Default | Description |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)